LogicLayer/Notifications/IMailSender.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GameBoard.LogicLayer.Notifications
+{
+    public interface IMailSender
+    {
+        Task SendEmailConfirmationAsync(string email, string link);
+        Task SendEventCancellationAsync(IEnumerable<string> emails, string link);
+        Task SendEventConfirmationAsync(IEnumerable<string> emails, string link);
+        Task SendEventInvitationAsync(IEnumerable<string> emails, string link);
+        Task SendFriendAcceptAsync(string email, string link);
```

Comment:
There is no such functionallity, we're not sending FriendAccept emails.

Reply:
Done